### PR TITLE
MVP Currency

### PR DESCRIPTION
**Pull request by boaz0**

### CODE REVIEW

File: koku/currency/models.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from django.db import models
```
Comment (dccurtis) [inline koku/currency/models.py:1]: FYI: We're going to want to make the migration additions in a separate PR to help simplify release rollbacks

Comment (boaz0) [reply koku/currency/models.py:1]: So should the migration PR be merged first or last?
